hugo/content/docs/data_seed.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Running Golden Seed Locally'
+date: 2021-07-21T12:12:25-05:00
+draft: true
+---
+
+# Golden Seed Implementation Steps
+
+1.  Start server: `rails s`
+
+2.  Start redis-server in another terminal. To do this run: `redis-server`
+
+        If redis is not found, run: ```brew install redis```
+
+3.  In a third terminal window, start sidekiq: `bundle exec sidekiq start`
+
+4.  \*Make sure there is enough memory on the server to run both of those commands
```

Comment:
there's an extra \ on this line, and you put a *before "make", a single astereik is for italicizing. Did you mean to put the single asteriks around "Make sure"? Maybe do that and get rid of the slash.